KomaMRICore/src/KomaMRICore.jl

```diff
@@ -13,7 +13,7 @@ using Interpolations
 using CUDA
 #Reconstruction
 using MRIBase, MRIFiles
-@reexport using MRIBase: Profile, RawAcquisitionData, AcquisitionData, AcquisitionHeader
+@reexport using MRIBase: EncodingCounters, Profile, RawAcquisitionData, AcquisitionData, AcquisitionHeader
 @reexport using MRIFiles: ISMRMRDFile
 #IO
```

Comment:
KomaCore should only use MRIBase.

Reply:
Addressed [here](https://github.com/cncastillo/KomaMRI.jl/commit/2150390776a4851662fd3cc2d4629cec61bc8a3b)